documentation/content/en/books/handbook/eresources/_index.adoc

```diff
@@ -176,25 +222,23 @@ If text/plain does not accompany text/html:
 [[eresources-news]]
 == Usenet Newsgroups
 
-In addition to two FreeBSD specific newsgroups, there are many others in which FreeBSD is discussed or are otherwise relevant to FreeBSD users.
+Alongside two FreeBSD-specific groups, other groups discuss FreeBSD or are otherwise relevant to users of FreeBSD.
 
-=== BSD Specific Newsgroups
+=== BSD-oriented Groups
 
-* link:news:comp.unix.bsd.freebsd.announce[comp.unix.bsd.freebsd.announce]
-* link:news:comp.unix.bsd.freebsd.misc[comp.unix.bsd.freebsd.misc]
-* link:news:de.comp.os.unix.bsd[de.comp.os.unix.bsd] (German)
-* link:news:fr.comp.os.bsd[fr.comp.os.bsd] (French)
+* link:news:comp.unix.bsd.freebsd.announce[comp.unix.bsd.freebsd.announce] -- FreeBSD-specific
+* link:news:comp.unix.bsd.freebsd.misc[comp.unix.bsd.freebsd.misc] -- FreeBSD-specific
```

Comment:
`comp.unix.bsd.freebsd.misc` visual overview: 

![comp unix bsd freebsd misc](https://github.com/freebsd/freebsd-doc/assets/192271/45ab3ed1-862e-4794-879e-41edca9744ad)

- thirteen on-topic threads this year
- overall, probably not bad enough to justify delisting.